interface/patient_tracker/patient_tracker.php

```diff
@@ -263,10 +263,10 @@ function openNewTopWindow(newpid,newencounterid) {
      <td colspan="12">
          <b><small>
              <?php
-             $statuses_output =  xlt('Total patients')  . ':' . $appointments_status['count_all'];
+             $statuses_output =  xlt('Total patients')  . ':' . text($appointments_status['count_all']);
              unset($appointments_status['count_all']);
              foreach($appointments_status as $status_symbol => $count){
-                 $statuses_output .= " | " . xlt($statuses_list[$status_symbol])  .":" . $count;
+                 $statuses_output .= " | " . text(xl_list_label($statuses_list[$status_symbol]))  .":" . $count;
```

Comment:
Sorry, I should of told you to use the xl_appt_category() last review:
``````
$statuses_output .= " | " . text(xl_appt_category($statuses_list[$status_symbol]))  .":" . $count;
``````